<launch>

  <param name="use_sim_time" value="false" />
  
  <!-- EDIT THIS LINE TO REFLECT THE NAME OF YOUR OWN MAP FILE 
       Can also be overridden on the command line -->
  <!--arg name="map" default="my_map_1.yaml" /-->

  <!-- Run the map server with the desired map -->
  <!--node name="map_server" pkg="map_server" type="map_server" args="$(find dashgo_nav)/maps/$(arg map)"/-->

  <!-- Fire up AMCL -->
  <!--include file="$(find dashgo_nav)/launch/teb_amcl_demo.launch" /-->

  <!-- Start the navigation test -->
  <node pkg="dashgo_nav" type="nav_test.py" name="nav_test" output="screen">
    <param name="rest_time" value="10" />
    <param name="fake_test" value="false" />
  </node>

</launch>
